src/plugins/auto/auto_executable_network.cpp

```diff
@@ -151,21 +151,20 @@ IE::Parameter AutoExecutableNetwork::GetMetric(const std::string& name) const {
         }
         return decltype(ov::optimal_number_of_infer_requests)::value_type {real};
     } else if (name == ov::execution_devices) {
+        if (_autoSchedule->_AutoCallMulti) {
```

Comment:
I still have concern with the necessity of this newly introduced member.
can we use something like this?
if ( _autoContext.perfhint as cumulative) {
  try { 
           exedevice = _autoSchedule->_loadContext[ACTUALDEVICE].executableNetwork->GetMetric(name);
}  catch {
          exedevice = _autoSchedule->_loadContext[ACTUALDEVICE].workName;
}

Reply:
Updated it.